src/core/lib/iomgr/ev_epollex_linux.c

```diff
@@ -122,27 +122,27 @@ static const char *polling_obj_type_string(polling_obj_type t) {
   return "<invalid>";
 }
 
-static char *pollable_desc(pollable_type *p) {
+static char *pollable_desc(pollable *p) {
   char *out;
   gpr_asprintf(&out, "type=%s group=%p epfd=%d wakeup=%d",
                polling_obj_type_string(p->po.type), p->po.group, p->epfd,
                p->wakeup.read_fd);
   return out;
 }
 
-static pollable_type g_empty_pollable;
+static pollable g_empty_pollable;
 
-static void pollable_init(pollable_type *p, polling_obj_type type);
-static void pollable_destroy(pollable_type *p);
+static void pollable_init(pollable *p, polling_obj_type type);
+static void pollable_destroy(pollable *p);
 /* ensure that p->epfd, p->wakeup are initialized; p->po.mu must be held */
-static grpc_error *pollable_materialize(pollable_type *p);
+static grpc_error *pollable_materialize(pollable *p);
 
 /*******************************************************************************
  * Fd Declarations
  */
 
 struct grpc_fd {
-  pollable_type pollable;
+  pollable pollable_obj;
```

Comment:
`pollable_obj` is great because it makes it clear that this is an object and not just a boolean condition.